packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -430,7 +433,7 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
                 highlightIndexNotifier: _highlightedOptionIndex,
                 child: Builder(
                   builder: (BuildContext context) {
-                    return widget.optionsViewBuilder(context, _select, _options);
+                    return widget.optionsViewBuilder(context, _select, _options, _fieldWidth ?? double.maxFinite);
```

Comment:
What if you try to get the width of the field here instead of always maintaining it?  Maybe you could get rid of the post frame callback.

Reply:
`Cannot get size during build`

Reply:
Ah true, even in an Overlay.